internal/postgresConfig/update/update.go

```diff
@@ -0,0 +1,67 @@
+package update
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/afero"
+	"github.com/supabase/cli/internal/postgresConfig/get"
+	"github.com/supabase/cli/internal/utils"
+	"strconv"
+	"strings"
+)
+
+func Run(ctx context.Context, projectRef string, values []string, replaceOverrides bool, fsys afero.Fs) error {
+	// 1. Prepare config overrides
+	newConfigOverrides := make(map[string]string)
+	for _, config := range values {
+		splits := strings.Split(config, ":")
+		if len(splits) != 2 {
+			return fmt.Errorf("expected config value in key:value format, received: '%s'", config)
+		}
+		newConfigOverrides[splits[0]] = splits[1]
+	}
+	// 2. If not in replace mode, retrieve current overrides
+	finalOverrides := make(map[string]interface{})
+	{
+		if !replaceOverrides {
+			config, err := get.GetCurrentPostgresConfig(ctx, projectRef)
+			if err != nil {
+				return err
+			}
+			finalOverrides = config
+		}
+	}
+	// 3. Create the list of final overrides
+	{
+		for k, v := range newConfigOverrides {
+			// this is hacky - if we're able to convert the value to an integer, we do so
+			// if we start supporting config fields with e.g. floating pt overrides this'll need to be updated
+			attemptedConvert, err := strconv.Atoi(v)
```

Comment:
This can be removed after next prod deploy.

Reply:
no - there's still ints that will throw an error